expression/bench_test.go

```diff
@@ -785,8 +785,8 @@ func genVecExprBenchCase(ctx sessionctx.Context, funcName string, testCase vecEx
 	}
 	cols := make([]Expression, len(testCase.childrenTypes))
 	input = chunk.New(fts, 1024, 1024)
-	for i, eType := range testCase.childrenTypes {
-		fillColumn(eType, input, i, testCase)
+	for i := range testCase.childrenTypes {
+		fillColumn(fts[i].EvalType(), input, i, testCase)
```

Comment:
Could you explain why this change?
This will make the `childrenTypes` enty of `testCase` useless.

Reply:
updated.